setup.py

```diff
@@ -1,8 +1,36 @@
 from setuptools import setup
+import os, subprocess
 
 with open('README.rst') as file:
     long_description = file.read()
 
+if not os.path.exists('.currently_generating_catalog'):
+    open('.currently_generating_catalog', 'ab').close()
+    subprocess.call('python setup.py compile_catalog', shell=True)
```

Comment:
This line I'm not sure about: it may requires babel as a development requirement.